### ISSUE TITLE
integrate GUI for Tournament

### ISSUE THREAD

**Issue by adangert** [adangert/JoustMania#57]
web ui currently does not work for Tournament mode

**Comment by adangert** [comment]
@ericbuzan would be cool if it could display something like this: http://www.aropupu.fi/bracket/

**Comment by ericbuzan** [comment]
Yeah that would look great. Once I get more free time I'll poke through the Tournament mode code and try that.

**Comment by martinbogo** [comment]
@ericbuzan Willing to collab on this... I have free time to work on Joust.
